guardian/self-assessment
# Recommend that users set up fine-grained access tokens

## What does this change?

Changes the instructions of the readme to suggest users set up fine-grained access tokens. The reasons for this are twofold

- Fine grained permissions typically result in tokens having a smaller overall permission set
- They enforce expiry dates. Users cannot have their credentials live forever, even if they want to

## How to test

Create a token using the steps and observe the results. Verify that the self assessment tool still works

## How can we measure success?

Users are still able to use the tool. Fewer tokens with lots of permissions and possibly no expiry date
